docs/update_and_migration/from_4.5/update_from_4.5.md

```diff
@@ -398,3 +398,56 @@ To update the schema, and then reindex the content, use the following commands:
 php bin/console ibexa:elasticsearch:put-index-template --overwrite
 php bin/console ibexa:reindex
 ```
+
+## v4.6.2
+
+#### Database update
+
+Run the following scripts:
+
+=== "MySQL"
+
+    ``` sql
```

Comment:
```suggestion
    ``` bash
```

We've been using `bash` styling for all SQL scripts in all other upgrade instructions until now. If you think `sql` styling is more fitting, please change it everywhere. Otherwise, use `bash`.

Reply:
You're right, this isn't SQL

Reply:
```suggestion
    ``` bash
```